examples/dt/uni01alpha/README.md

```diff
@@ -299,6 +301,139 @@ spec:
           octavia: octbr
 ```
 
+### Ironic
+
+Ironic service is enabled and appropriate network configuration is applied
+to configure Baremetal-as-a-Service. It manages baremetal nodes and enables
+baremetal nova instances.
+
+A dedicated network is required for baremetal node management, such as
+provisioning, cleaning and inspection. This network should be connected
+to the OCP worker nodes, networker node and the test VMs (Virtual Baremetal).
+
+The appropriate bridge mappings must be configured in Neutron (OVN).
+
+```yaml
+spec:
+  ironic:
+    enabled: true
+    template:
+      databaseInstance: openstack
+      ironicAPI:
+        override:
+          service:
+            internal:
+              metadata:
+                annotations:
+                  metallb.universe.tf/address-pool: internalapi
+                  metallb.universe.tf/allow-shared-ip: internalapi
+                  metallb.universe.tf/loadBalancerIPs: 172.17.0.80
+              spec:
+                type: LoadBalancer
+        replicas: 1
+      ironicConductors:
+      - customServiceConfig: |
+          [neutron]
+          cleaning_network = provisioning
+          provisioning_network = provisioning
+          rescuing_network = provisioning
+        networkAttachments:
+        - ironic
+        provisionNetwork: ironic
+        replicas: 1
+        storageRequest: 10G
+      ironicInspector:
+        inspectionNetwork: ironic
+        networkAttachments:
+        - ironic
+        replicas: 1
+      ironicNeutronAgent:
+        replicas: 1
+      secret: osp-secret
+
+  nova:
+    template:
+      cellTemplates:
+        cell1:
+          cellDatabaseAccount: nova-cell1
+          cellDatabaseInstance: openstack-cell1
+          cellMessageBusInstance: rabbitmq-cell1
+          hasAPIAccess: true
+          novaComputeTemplates:
+            compute-ironic:
+              computeDriver: ironic.IronicDriver
+
+  ovn:
+    template:
+      ovnController:
+        nicMappings:
+          datacentre: ocpbr
+          ironic: ironic
+```
+
+#### Ironic cloud configuration
+
+##### Provisioning network
+
+```bash
+openstack network create provisioning \
+  --share --provider-physical-network ironic \
+  --provider-network-type flat \
+  --availability-zone-hint zone-1
+openstack subnet create provisioning-subnet \
+  --network provisioning \
+  --subnet-range 172.20.254.0/24 \
+  --gateway 172.20.254.1 \
+  --allocation-pool start=172.20.254.100,end=172.20.254.200
+```
+
+##### Nova flavor
+
```

Comment:
I guess there is a TODO to define a flavor that targets the ironic nodes

Reply:
Yes, we can add that. (But I will first try to figure out what we need to do to get tempest passing.)

Reply:
Added example command to create flavor.